.github/workflows/publish.yml

```diff
@@ -1,7 +1,7 @@
 on:
   workflow_dispatch:
   push:
-    branches: [main]
+    branches: [169-bug-calling-appendix-fix-echo]
```

Comment:
is this intended?

Reply:
Yeah I do that to check whether rendering works without having to merge into main. 
Thanks for checking, it is removed now :)